docusaurus/docs/Android/04-compose/04-message-components/05-attachments-picker.mdx

```diff
@@ -67,12 +67,12 @@ AttachmentsPicker(
     ..., // State and UI customization
     onAttachmentsSelected = { attachments -> 
         // Dismiss the picker and store the attachments
-        attachmentsPickerViewModel.onShowAttachments(false)
-        composerViewModel.onAttachmentsSelected(attachments)
+        attachmentsPickerViewModel.changeAttachmentState(false)
```

Comment:
`changeAttachmentState(false)` is quite cryptic, could either use a better method name or have a named parameter at the call site here

Reply:
Will add named arguments!